### PR TITLE
Limit GITHUB_TOKEN permissions

### PR DESCRIPTION
**Pull request by jedsalazar**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -9,10 +9,20 @@ on:
...
   golangci:
+    permissions:
+      contents: read  # for actions/checkout to fetch code
+      pull-requests: read  # for golangci/golangci-lint-action to fetch pull requests
     runs-on: ubuntu-latest
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        with:
+          egress-policy: audit
```
Comment (cpanato) [inline .github/workflows/lint.yaml:25]: as you are changing this can you bump the golangci-lint version to v1.56?

**Review by cpanato — APPROVED** [review]
check my comment but it is a non blocking

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (jedsalazar) [reply .github/workflows/lint.yaml:25]: done!

**Review by cpanato — APPROVED** [review]
thx

**Pull request merged**